x/programs/wasmlanche/src/context.rs

```diff
@@ -40,29 +41,37 @@ mod debug {
     impl Debug for Context {
         fn fmt(&self, f: &mut Formatter<'_>) -> Result {
             let Self {
-                program,
+                contract_address,
```

Comment:
Fine to leave this for now, but I'm thinking that we should call this `state_address` in the future. The contract code may change and while the code won't actually have an address, I think it would be good to think of contract execution as combining a state-address with a bytecode-id (hash) to perform execution